docs/faq.md

```diff
@@ -8,34 +8,32 @@ weight: 7
 
 #### Table of Contents
 
-         * [Frequently Asked Questions](#frequently-asked-questions)
-            * [Table of Contents](#table-of-contents)
-            * [What is KUDO?](#what-is-kudo)
-            * [When would you use KUDO?](#when-would-you-use-kudo)
-            * [What is a Framework](#what-is-a-framework)
-            * [What is a FrameworkVersion?](#what-is-a-frameworkversion)
-            * [What is an Instance?](#what-is-an-instance)
-            * [What is a Plan?](#what-is-a-plan)
-            * [What is a PlanExecution?](#what-is-a-planexecution)
-            * [What is a Parameter?](#what-is-a-parameter)
-            * [What is a Strategy?](#what-is-a-strategy)
-            * [What is a Trigger?](#what-is-a-trigger)
-            * [When I create a Framework, it is automatically creating new CRDs?](#when-i-create-a-framework-it-is-automatically-creating-new-crds)
-               * [How does it work from a RBAC perspective?](#how-does-it-work-from-a-rbac-perspective)
-            * [Is the dependency model an individual controller per workload?](#is-the-dependency-model-an-individual-controller-per-workload)
-
-#### What is KUDO?
+* [What is KUDO?](#what-is-kudo)
+* [When would you use KUDO?](#when-would-you-use-kudo)
+* [What is a Framework](#what-is-a-framework)
+* [What is a FrameworkVersion?](#what-is-a-frameworkversion)
+* [What is an Instance?](#what-is-an-instance)
+* [What is a Plan?](#what-is-a-plan)
+* [What is a PlanExecution?](#what-is-a-planexecution)
+* [What is a Parameter?](#what-is-a-parameter)
+* [What is a Deployment Strategy?](#what-is-a-deployment-strategy)
+* [What is a Trigger?](#what-is-a-trigger)
+* [When I create a Framework, will it automatically create new CRDs?](#when-i-create-a-framework-will-it-automatically-create-new-crds)
+   * [How does it work from a RBAC perspective?](#how-does-it-work-from-a-rbac-perspective)
+* [Is the dependency model an individual controller per workload?](#is-the-dependency-model-an-individual-controller-per-workload)
+
+#### **What is KUDO?**
```

Comment:
Instead of adding bold to the headers remove one pound signs so we get `h3` instead of `h4`

Reply:
`h2` looks better... thanks!